docs/source/features/parallelisms.rst

```diff
@@ -5,15 +5,80 @@ Parallelisms
 
 NeMo Megatron supports 5 types of parallelisms (which can be mixed together arbitrarily):
 
+Data Parallelism
+^^^^^^^^^^^^^^^^
+
+Data Parallelism (DP) creates identical copies of the model across
+multiple GPUs. Data batches are distributed between GPUs so that the
+GPUs can process them independently. While compute is efficiently
+distributed between GPUs, communication is required in order to keep
+the model copies consistent with each other.
+
 Distributed Data Parallelism
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Distributed Data Parallelism (DDP) creates idential copies of the model across multiple GPUs.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Distributed Data Parallelism (DDP) keeps model copies consistent by
+synchronizing parameter gradients before each optimization step. More
+specifically it sums gradients over all model copies using an
+all-reduce communication collective.
 
 .. image:: ../nlp/nemo_megatron/images/ddp.gif
     :align: center
     :width: 800px
     :alt: Distributed Data Parallel
 
+Distributed Optimizer (ZeRO-1)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ZeRO-1 algorithm keeps model copies consistent by sharding the
+optimizer state between GPUs. During each optimization step, the
+parameter gradients are first summed and sharded (with a
+reduce-scatter collective), each GPU applies an optimization to its
+local shard of the parameters, and the updated parameter shards are
+broadcast to update all of the model copies (with an all-gather
+collective). This approach is attractive for large models since
+sharding the optimizer state can significantly reduce its memory
+footprint on individual GPUs. It also has, in theory, the same
+communication volume as DDP and its communication pattern has more
+opportunities for overlapping with compute.
+
+Enable Data Parallelism
+~~~~~~~~~~~~~~~~~~~~~~~
+
+DDP is the default parallelism scheme when NeMo is run on multiple
+GPUs. Enabling other parallelism schemes in the model configuration
+will decrease the size of the DP group, i.e. the number of identical
```

Comment:
Change "will decrease the size of the DP group, i.e. the number of identical . . ." to "will decrease the size of the DP group, for example, the number of identical . . .